mne/viz/utils.py

```diff
@@ -1793,7 +1794,7 @@ class SelectFromCollection(object):
     """
 
     def __init__(self, ax, collection, ch_names,
-                 alpha_other=0.3):  # noqa: D102
+                 alpha_other=0.3):  # noqa: D107
```

Comment:
D107 is in our global ignore list so no need to add it:

https://github.com/mne-tools/mne-python/blob/master/setup.cfg#L45